conda/recipes/numbast_extensions/meta.yaml

```diff
@@ -17,13 +18,32 @@ source:
   path: ../../../numbast_extensions
 
 build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - AWS_ACCESS_KEY_ID
+    - AWS_SECRET_ACCESS_KEY
+    - AWS_SESSION_TOKEN
+    - CMAKE_C_COMPILER_LAUNCHER
+    - CMAKE_CUDA_COMPILER_LAUNCHER
+    - CMAKE_CXX_COMPILER_LAUNCHER
+    - CMAKE_GENERATOR
+    - PARALLEL_LEVEL
+    - SCCACHE_BUCKET
+    - SCCACHE_IDLE_TIMEOUT
+    - SCCACHE_REGION
+    - SCCACHE_S3_KEY_PREFIX=numbast-aarch64 # [aarch64]
+    - SCCACHE_S3_KEY_PREFIX=numbast-linux64 # [linux64]
+    - SCCACHE_S3_USE_SSL
+    - SCCACHE_S3_NO_CREDENTIALS
   script:
     - {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - python
     - pip
+    - setuptools
```

Comment:
ditto

Reply:
Same, I think it is just becoming explicit and doesn't hurt in older versions where it was once implicit.